notebooks/Model_Training_with_TensorArray.ipynb

```diff
@@ -0,0 +1,5826 @@
+{
```

Comment:
<p>In <code>spans_df = spans_df.reindex(columns=mns=["span", "token_span", "ent_type"])</code>  is it supposed to be <code>columns=mns=...</code>?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/CODAIT/text-extensions-for-pandas/pull/116/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='16'/>